ext/bindings/consumer.cpp

```diff
@@ -58,6 +59,18 @@ void bind_consumer(Module &module) {
     //.define_method("negative_acknowledge", &pulsar_rb::Consumer::negative_acknowledge)
     ;
 
+  Enum<pulsar::ConsumerType> rb_cConsumerType =
+    define_enum<pulsar::ConsumerType>(
+      "ConsumerType",
+      module
+    );
+
+  rb_cConsumerType
```

Comment:
optional: no need to capture the intermediate rb_cConsumerType variable, as it's not used in more than this one place after creation. can just do
```
define_enum<...>(...)
  .define_value(...)
  .define_value(...)
// etc.
```
similar to the preceding and following `define_class_under` chunks.

but doesn't really hurt to have, so not a blocker